Add support for deploying SSP Installer OVA with pod

rexit1982 opened the pull request:
Enabling deploymet of ssp installer ova with a pod, have been using in my lab for a couple weeks.  Will want to add extra FQDNs for the SSP items and reserve pod IPs at some point.

rexit1982 committed:
add support for ssp installer

rexit1982 committed:
merge ssp to dev-v8

luischanu commented:
@rexit1982 , in looking at the commits, it appears that you might have accidentally duplicated the `Nested_SDDCManager` section in `playbooks/UpdateDNS.yml` playbook.   Can you double-check that?

Also, I noticed that `playbooks/CreatePodDoc.yml` playbook wasn't updated.  Could you update that playbook to document the new SSPI product?

Thanks,

Luis

luischanu commented:
@rexit1982 , I also just noticed an issue with the config_sample.yml.   It appears as though you didn't update the IPv6 address correctly, as it appears as though you're assigning 26 rather than 28 for the IPv6.   Can you double-check that, too?

Thanks,

Luis

rexit1982 committed:
fix duplicated DNS update and ipv6 addr for sspi

rexit1982 commented:
It looks like I did duplicate the SDDCM piece and had a couple references to it that got missed in echos for other parts.

CreatePod Doc didn't need to be updated, that is all handled in the PodDoc j2 template that I did catch myself on before I submitted.

I also missed the ipv6 thing, was thinking that was a character length and not translation.  Have it fixed in a moment.

luischanu commented:
@rexit1982 , yes, just the PodDoc.j2 needed updating, not the playbook.  I should have been more clear about that.  Sounds good on the other items.

rexit1982 commented:
@luischanu Yes, the PodDoc.j2 files has already been updated for SSPI see [Diff Here](https://github.com/rutgerblom/SDDC.Lab/pull/490/files#diff-410bf2bd07f4f41def66228d50485c231539a363c360d6c93df45799eec4a353)

luischanu commented:
@rexit1982 I haven’t checked all the commits, but are you finished correcting the small items that we brought to your attention on your initial PR?  We just want to make sure things are all “good to go” before we roll in your commits.  Please advise…

Thanks!

Luis

luischanu commented:
@rexit1982 , it looks like there is an issue with SSPI when creating a new PodConfig.  Here is the error I get when I run CreatePodConfig.yml playbook:
![image](https://github.com/user-attachments/assets/426a1ebb-4600-4b1b-a190-d673e6939908)

I'm looking into it now...

rexit1982 commented:
Hmm, I hit that during development.  Double check your config updated all
the places.

On Tue, Mar 18, 2025, 7:15 PM Luis Chanu ***@***.***> wrote:

> @rexit1982 <https://github.com/rexit1982> , it looks like there is an
> issue with SSPI when creating a new PodConfig. Here is the error I get when
> I run CreatePodConfig.yml playbook:
> image.png (view on web)
> <https://github.com/user-attachments/assets/426a1ebb-4600-4b1b-a190-d673e6939908>
>
> I'm looking into it now...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734933256>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZ32E3NQCJX4VN4HQQ63HT2VCSHVAVCNFSM6AAAAABZBO2ZGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUHEZTGMRVGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: luischanu]*luischanu* left a comment (rutgerblom/SDDC.Lab#490)
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734933256>
>
> @rexit1982 <https://github.com/rexit1982> , it looks like there is an
> issue with SSPI when creating a new PodConfig. Here is the error I get when
> I run CreatePodConfig.yml playbook:
> image.png (view on web)
> <https://github.com/user-attachments/assets/426a1ebb-4600-4b1b-a190-d673e6939908>
>
> I'm looking into it now...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734933256>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZ32E3NQCJX4VN4HQQ63HT2VCSHVAVCNFSM6AAAAABZBO2ZGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUHEZTGMRVGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

luischanu commented:
@rexit1982  Yep, I found the issue, which is why I deleted the comment.  Sorry about that.  Thanks!

rexit1982 commented:
Glad to hear it!
It took me WAY too long to figure that out when I ran into it....I was
verifying in the config sample file not my to build config.

On Tue, Mar 18, 2025 at 7:24 PM Luis Chanu ***@***.***> wrote:

> @rexit1982 <https://github.com/rexit1982> Yep, I found the issue, which
> is why I deleted the comment. Sorry about that. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734943239>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZ32E3XB3HEWHFRYUECI332VCTJXAVCNFSM6AAAAABZBO2ZGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUHE2DGMRTHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: luischanu]*luischanu* left a comment (rutgerblom/SDDC.Lab#490)
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734943239>
>
> @rexit1982 <https://github.com/rexit1982> Yep, I found the issue, which
> is why I deleted the comment. Sorry about that. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rutgerblom/SDDC.Lab/pull/490#issuecomment-2734943239>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZ32E3XB3HEWHFRYUECI332VCTJXAVCNFSM6AAAAABZBO2ZGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUHE2DGMRTHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
